bigtable/client/table_admin_test.cc

```diff
@@ -306,6 +306,74 @@ TEST_F(TableAdminTest, CreateTableFailure) {
 #endif  // GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
 }
 
+/**
+ * @test Verify that Copy Constructor and assignment operator
+ * copies all properties.
+ */
+TEST_F(TableAdminTest, CopyConstructibleAssignableTest) {
+  using namespace ::testing;
```

Comment:
Same thing, add static assertions here on in `table_admin.cc`.

Reply:
I do not see the static asserts for `bigtable::TableAdmin` (without `noex`) did I miss them?